repose-aggregator/components/filters/saml-policy-translation-filter/src/test/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilterTest.scala

```diff
@@ -108,7 +164,51 @@ class SamlPolicyTranslationFilterTest extends FunSpec with BeforeAndAfterEach wi
   }
 
   describe("determineVersion") {
-    pending
+    val document = DocumentBuilderFactory.newInstance()
+      .newDocumentBuilder()
+      .parse(new InputSource(new StringReader(samlResponse)))
+
+    def buildConfig(issuer: String): SamlPolicyConfig = {
+      val resultConfig = new SamlPolicyConfig
+      val bypassIssuers = new PolicyBypassIssuers
+      bypassIssuers.getIssuer.add(issuer)
+      resultConfig.setPolicyBypassIssuers(bypassIssuers)
+      val acquisition = new PolicyAcquisition
+      val cache = new Cache
+      cache.setTtl(300)
+      acquisition.setCache(cache)
+      resultConfig.setPolicyAcquisition(acquisition)
```

Comment:
If this comes in after REP-4809, then it will need a `resultConfig.setSignatureCredentials(signatureCredentials)` added for the `filter.configurationUpdated(config)` to not error out.